src/components/views/ScreeningsSection.vue

```diff
@@ -37,12 +36,37 @@ export default defineComponent({
       }
       return nextDays
     },
+    activeDayDate() {
+      const activeDayIndex = this.nextDaysNames.indexOf(this.activeDay)
+      const today = new Date()
+      let activeDay = new Date()
+      activeDay.setDate(today.getDate() + activeDayIndex)
+      return activeDay
+    },
+    formattedActiveDayDate() {
+      const date = this.activeDayDate.toISOString().slice(0, 10)
+      return date.split('-').reverse().join("/")
+    },
+    activeDayName() {
+      return this.activeDayDate.toLocaleString('en-GB', {weekday: 'long'});
+    },
+    sectionSubtitle() {
+      return `${this.activeDayName} ${this.formattedActiveDayDate}`;
+    },
+    moviesTitles() {
+      const movieTitles = []
+      for (let movie of this.movies) {
+        movieTitles.push(movie.title)
+      }
+      return movieTitles
```

Comment:
Use the power of the map here 😎 
```suggestion
      return this.movies.map(movie => movie.title);
```

Reply:
I must have packed `map` and `filter` methods to my React mental drawer -- I completely forgot to use them here 😆